internal/maincmd/namespacing_linux.go

```diff
@@ -15,6 +14,7 @@ import (
 	"syscall"
 
 	"github.com/gokrazy/rsync/rsyncd"
+	"github.com/gokrazy/rsync/internal/log"
```

Comment:
This file does not seem to be `gofmt`ed? (Makes the CI fail.)

The rsyncd line goes below the internal line.